Model.php

```diff
@@ -194,6 +194,23 @@ public function getTriggeredAlerts($idSites, $login)
         return $alerts;
     }
 
+    public function getTriggeredAlertsFromPastNHours(string $period, int $idSite, int $hours)
+    {
+        $timestamp = Date::now()->addHour(-1 * $hours)->getDatetime();
+
+        $db  = $this->getDb();
+        $sql = $this->getTriggeredAlertsSelectPart()
```

Comment:
@snake14 Cane we break this function and add a test for that function ?
Like what SQL query it generates

Reply:
@AltamashShaikh That's a good idea, but seems a bit excessive when this improvement is already over estimate. If you're talking about the `getTriggeredAlertsSelectPart`, that's an existing method that took advantage of. Overall, it's a pretty basic query.

Reply:
Yaa that's okay :+1: